.github/dependabot.yml

```
@@ -1,44 +1,58 @@
 version: 2
 updates:
+  # biweekly minor and patch
   - package-ecosystem: npm
     target-branch: dev
-    directory: '/'
+    directory: "/"
     schedule:
-      interval: weekly
-      day: monday
-      time: '07:00'
-      timezone: 'Europe/Berlin'
+      interval: cron
+      cronjob: "0 8 * * mon%2"
+      timezone: "Europe/Berlin"
+    groups:
+      yarn-minor-patch:
```

Couldn't we have all unified instead of 2 `package-ecosystem: npm`? Like 
```suggestion
      yarn-minor-patch:
         update-types: ["patch", "minor"]
      yarn-major:
        update-types: ["major"]
      
```

---

It work for a short time on Friday 🙃